### ISSUE TITLE
Outbound "Email Notification" configuration via port 465 defect

### ISSUE THREAD

**Issue by fthommen** [zammad/zammad#1084]
### Infos:

* Used Zammad version: 1.5.0
* Used Zammad installation source: DEB
* Operating system: Debian 8
* Browser + version: Not relevant

### Problem Description

Configuring authenticated outbound mail via SMTP on port 465 (SSL) works fine for Channels -> Email -> Email Accounts -> Outbound, but the identical settings generate an error "No Such User" when used to configure Channels -> Email -> Email Notification -> Outbound (see attached screenshot):

![zammad_smtp_outbound_error](https://cloud.githubusercontent.com/assets/279232/26038320/1b8236ae-3906-11e7-802b-a5cfde8265ce.png)

When deliberately entering a wrong password in the configuration form, the automatic configuration testing issue an "Authentication error" error, which seems to show, that the account indeed _does_ exist.

There seem to be two problems:

a) the form for the Email Notification configuration doesn't seem to work properly

b) the two forms for outbound mail (for Email Accounts and for Email Notification) don't seem to use the same codebase

**Comment by fthommen** [comment]
Hello, is there a - quick - solution for this bug in sight?  Preferrably still in Zammad 1.5 ;-). This issue is kind of a blocker for us, as our mailrelays don't allow sending emails to external addresses via port 25.  Since SMTP on port 465 works fine for the "Email Accounts" settings, why not just use the same code for the "Email Notification"?

**Comment by dennybresch** [comment]
Hello @fthommen ,

we are using the same module / codebase for the email account configuration and the email notification configuration.
Furthermore, we could not reproduce this issue yet.

Is there the possibility to get the logfile from the mailserver?

The "Notification Sender" may have to be configured.
Please check this via:
Channels -> Emails -> Settings -> Notification Sender

Best regards,
Denny

**Comment by fthommen** [comment]
> The "Notification Sender" may have to be configured.
> Please check this via:
> Channels -> Emails -> Settings -> Notification Sender

That was it!  There was a notification sender configured, but it was a noreply-address which didn't exist on the SMTP server (tested both with a foreign domain and with the local domain) and therefore the server refused to relay the mail (even though we authenticated as a valid user).  By using the ticketing system's main address we finally succeeded to configure the email notification.  Thanks a lot.
